catalogs/catalog-hive/src/test/java/com/datastrato/gravitino/catalog/hive/TestDownloadUtils.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+
+package com.datastrato.gravitino.catalog.hive;
+
+import java.io.File;
+import org.apache.hadoop.conf.Configuration;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class TestDownloadUtils {
+
+  @Test
+  public void testDownloadFile() throws Exception {
+
+    File srcFile = new File("test");
+    File destFile = new File("dest");
+
+    srcFile.createNewFile();
+    DownloadUtils.downloadFile(srcFile.toURI().toString(), destFile, 10, new Configuration());
+    Assertions.assertTrue(destFile.exists());
+
+    srcFile.delete();
```

Comment:
It's best to test one remote file system to ensure it functions properly.

Reply:
Do you mean that I should add a MiniHdfsCluster for this pr?

Reply:
Fixed.